exonum-light-client/src/main/java/com/exonum/client/ExonumClient.java

```diff
@@ -108,42 +108,45 @@
 
   /**
    * Returns blockchain blocks information for the requested range. The blocks are returned
-   * in reverse order, starting from the {@code heightMax}.
-   * @param count Number of blocks to return.
-   *        It should be in range [1, {@linkplain ExonumApi#MAX_BLOCKS_PER_REQUEST}]
-   * @param blockFilter controls whether to skip blocks with no transactions
-   * @param heightMax maximum height of the returned blocks.
+   * in ascending order by their height.
```

Comment:
Should we describe that it can lead to performance degradation in case of large ranges?

Reply:
I am actually not sure what we shall convey here. The thing that it takes space proportional to the number of blocks sounds obvious and reasonable. No one would reasonably expect anything different. Same goes for the time it takes to fetch them :thinking: